romtools/__init__.py

```diff
@@ -125,11 +125,14 @@ def getNumVars(self) -> int:
     snapshots = HeatSnapshots(data)
 ```
 
-
-
+# License
+```plaintext
+.. include:: ../LICENSE
+```
 '''
 
-__docformat__ = "markdown"  # explicitly disable rST processing in the examples above.
+__docformat__ = "restructuredtext"
+# __docformat__ = "markdown"  # explicitly disable rST processing in the examples above.
```

Comment:
please remove the markdown line and add a comment to explain why we need rst

Reply:
Done